taworks/taform/templates/taform/base.html

```diff
@@ -0,0 +1,27 @@
+<html>
+<header id = "banner"></header>
+<head>
+</head>
+<body>
+{% if AC %}
+<table align="right">
+	<tr>
+		<td>
+			<form>
+		        <button class="AC_button", formaction="{% url 'home' %}">Home</button>
```

Comment:
can we align the code spaces up on this page?

Reply:
I think its just Github being weird with the display:
![screen shot 2018-02-11 at 3 40 52 pm](https://user-images.githubusercontent.com/22134649/36078255-1974d700-0f42-11e8-9bed-8264f3956dfd.png)